eth/backend.go

```diff
@@ -553,13 +553,20 @@ func New(ctx context.Context, stack *node.Node, config *ethconfig.Config, logger
 			roTxLimit := int64(stack.Config().Http.DBReadConcurrency)
 
 			bridgeConfig := bridge.Config{
```

Comment:
Majority of fields here are from borConfig. Why not embed borConfig into bridgeConfig?